src/styles/blocks/_card.scss

```diff
@@ -0,0 +1,42 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
```

Comment:
card should have padding for all sides. So you'll avoid using left/right margin for children